packages/builder/builder-cli/package.json

```diff
@@ -37,7 +37,7 @@
     "new": "modern new",
     "build": "modern-lib build",
     "dev": "modern-lib build --watch",
-    "test": "vitest run",
+    "test": "vitest run --no-coverage",
```

Comment:
Why not just remove https://github.com/web-infra-dev/modern.js/blob/main/scripts/vitest-config/src/index.ts#L8-L10

Reply:
I did not notice that.

Reply:
Haha it's a bit hidden